crates/hyperqueue/src/client/output/json.rs

```diff
@@ -251,6 +251,10 @@ impl Output for JsonOutput {
     fn print_error(&self, error: Error) {
         self.print(json!({ "error": format!("{error:?}") }))
     }
+
+    fn finalize_output(&self) {
+        self.print(json!({}));
```

Comment:
This should only be printed if we did not print anything in the JSON printer. Otherwise we will print multiple JSON objects, which is invalid JSON.